docs/general/events.md

```diff
@@ -32,17 +32,74 @@ field semantics, and stability and requirement levels. Other events may be user-
+    the _payload_ (data).
+  * Whether there are any expected [standard](https://github.com/open-telemetry/opentelemetry-specification/tree/v1.31.0/specification/common#attribute)
+    attributes that are used to describe the event.
+* All OpenTelemetry defined events SHOULD be defined in the semantic conventions
+  to highlight the expected structure of the event.
+* The _payload_ (data) MAY contain any type supported by the OpenTelemetry data
+  model for the log [body](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/logs/data-model.md#field-body)
+  and the semantic conventions will define the expected structure of the _payload_
+  (data) for the event.
+* The _payload_ (data) SHOULD be used to represent the structure of the event.
+
+Recommendations for defining events:
+
+* Use the _payload_ (data) to represent the details of the event instead of a
+  collection of [standard](https://github.com/open-telemetry/opentelemetry-specification/tree/v1.31.0/specification/common#attribute)
+  attributes.
+* Events SHOULD be generated / produced / recorded using the
+    [Event API](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/logs/event-api.md)
+    to ensure that the event is created using the configured SDK instance.
+  * The Event API is not yet available in all OpenTelemetry SDKs.
+  * TODO: Add deep link to the [compliance matrix of the Event API](https://github.com/open-telemetry/opentelemetry-specification/blob/main/spec-compliance-matrix.md)
+    when it exists.
+* The _payload_ (data) _fields_ SHOULD not be prefixed with the `event.name` to
+  avoid redundancy and to keep the event definition clean.
 
-## General event attributes
+### Event payload (data)
 
-Events are recorded as LogRecords that are shaped in a special way: Event
-LogRecords MUST have the attribute `event.name` that uniquely identifies the event.
-Events with the same `event.name` are structurally similar to one another. Events
-may also have other LogRecord attributes.
+* Common attribute naming conventions and [registry](../attributes-registry/README.md)
+  requirements don't apply to event payload fields.
+* The definition for OpenTelemetry defined events supports describing
+  individual _fields_ (Body Fields)
+  * Any _fields_ are NOT added to the Global attribute registry for use by other
+    events.
+  * The _fields_ are unique to the named event (`event.name`) and different events
+    may use the same _field_ name to represent different data, due to the unique
+    nature of the event.
+* The _fields_ MAY reference / inherit details from Global attribute registry
+  attributes and provide additional details specific to the event, including
+  providing an _alias_ (shorter) name for the attribute.
+
+As each `event.name` is unique and defines the structure of the event, as long
+as the event follows the General event semantics it will be considered a valid
+event. For example :
```

Comment:
> As each `event.name` is unique and defines the structure of the event, as long
as the event follows the General event semantics it will be considered a valid
event.

Does the exception logs referenced below demonstrate something related to this sentence? 

Also, what information do we share in this sentence? A definition of the valid event? But we already defined it above.
Unless there is a notion of valid or invalid event or some handling that needs to happen for invalid ones, I suggest removing this sentence.

Reply:
Yeah, I think we can clarify this later if needed, as it not really adding much as it is today.

In terms of "invalid" events, its really going to be up to each event definition on what / how it should be handled (if) an event introduces breaking changes within it's own domain.